notebooks/MAST/astroqueryMAST/beginner_program_id_search/companion_script.py

```diff
@@ -0,0 +1,48 @@
+import sys
+from astroquery.mast import Observations
+
+def download_by_jwst_pid (pid, curl_flag):
+    """
+    This function retrieves all of the level 2 science products for a single program ID.
+    Inputs:
+    pid [int] – program ID number
+    curl_flag [Boolean] – If set to true, function returns a curl script.
+    """
+
+    # Login, search for observations
+    Observations.login(store_token=True)
```

Comment:
Do we need additional text / a pointer to instruct people how to make a token if they don't already have one?  Will this work as-is if they don't have one created at all yet?